packages/webchat/test/index.html

```diff
@@ -5,7 +5,10 @@
     <title>Webchat</title>
 
     <link rel="stylesheet" href="../../components/src/css/botpress-default.css" />
-    <link rel="stylesheet" href="../../../node_modules/@blueprintjs/core/lib/css/blueprint.css" />
+    <link
+      rel="stylesheet"
+      href="../../.././.yarn/cache/@blueprintjs-core-npm-3.52.0-8e614f339f-e0d46e0446.zip/node_modules/@blueprintjs/core/lib/css/blueprint.css"
```

Comment:
I think you should unplug blueprint instead of doing this: https://yarnpkg.com/cli/unplug

Reply:
not yet